.github/workflows/test.yml

```diff
@@ -37,6 +37,26 @@ jobs:
           docker-compose build
           docker-compose push test
 
+  docs_lint:
```

Comment:
This job doesn't need the application, so we can do it without the Docker image.
You can check [this workflow](https://github.com/nimblehq/infrastructure-templates/blob/develop/.github/workflows/lint.yml), and just change form npm to yarn 🙏

Reply:
Fixed in https://github.com/nimblehq/nimble-survey-web/pull/91/commits/02987118870df2fc227a951d6d912017d88ffa3e